deploy/charts/rook-ceph/values.yaml

```diff
@@ -567,6 +567,11 @@ cephCommandsTimeoutSeconds: "15"
 # -- if true, run rook operator on the host network
 useOperatorHostNetwork:
 
+# -- if true, scale down the rook operator
```

Comment:
nit: The generated doc doesn't read smoothly without a new sentence.
```suggestion
# -- If true, scale down the rook operator.
```

Reply:
Thanks @travisn - I've applied this fix